Update ReadMe (#228)
NilFoundation/placeholder: proof-producer/readme.md

# Proof producer
Executable for generating and verifying zk proof.

> Supported single end multi threaded versions.

Reviewer: typo: and

Author: done

## Contents
1. [Structure](#structure)
2. [How to use](#how_to_use)
3. [Building from source](#building_from_source)
4. [Sample calls to proof-producer](#sample_calls_to_proof-producer)

## Structure
proof-producer
├── bin
├── cmake
├── libs
└── tests

## How to use

> The input for the proof producer is a EVM trace or pair circuit and assignment table.

Reviewer: ```suggestion
The input for the proof producer is an EVM trace or a circuit plus an assignment table.
```

Author: done
